baseplate/__init__.py

```diff
@@ -93,8 +96,90 @@ def make_tracing_client(raw_config, log_if_unconfigured=True):
     )
 
 
+def crash_reporter_from_config(raw_config, module_name):
```

Comment:
Maybe it's because I haven't dealt with sentry a lot, but what is ``module_name`` being used for here?  Is it just what will be reported back in the error message?  Or will it be used to generate extra data like the surrounding lines for display on the web interface?  Should ``__name__`` always be used as the argument?